Platform/ElkhartlakeBoardPkg/AcpiTables/Dsdt/PchSerialIoSpi.asl

```diff
@@ -41,7 +41,6 @@ Scope(\_SB.PC00) {
 //-------------------------------------------
   Device (SPI0) {
     If(LEqual(SPI0_MODE, SERIAL_IO_SPI_HIDDEN)) {
-      Name (_HID, "PNP0C02")
```

Comment:
if _HID is removed,  how OS will know what type of device it is ?

Reply:
From the ACPI specification:
"A device object must contain either an _HID object or an _ADR object, but must not contain both." and this is the warning reported form iasl compiler.
I think its fine to remove because its referring to SPI HIDDEN MODE and no OS interaction will happen.